spec/timezone.html

```diff
@@ -626,9 +626,15 @@ <h1>BuiltinTimeZoneGetInstantFor ( _timeZone_, _dateTime_, _disambiguation_ )</h
       <h1>GetPossibleInstantsFor ( _timeZone_, _dateTime_ )</h1>
       <emu-alg>
         1. Let _possibleInstants_ be ? Invoke(_timeZone_, *"getPossibleInstantsFor"*, « _dateTime_ »).
-        1. Let _list_ be ? CreateListFromArrayLike(_possibleInstants_, « Object »).
-        1. For each element _instant_ in _list_ in List order, do
-          1. Perform ? RequireInternalSlot(_instant_, [[InitializedTemporalInstant]]).
+        1. Let _iteratorRecord_ be ? GetIterator(_possibleInstants_, ~sync~).
+        1. Let _list_ be a new empty List.
+        1. Let _next_ be *true*.
+        1. Repeat, while _next_ is not *false*,
+          1. Set _next_ to ? IteratorStep(_iteratorRecord_).
+          1. If _next_ is not *false*, then
+            1. Let _nextValue_ be ? IteratorValue(_next_).
+            1. Perform ? RequireInternalSlot(_nextValue_, [[InitializedTemporalInstant]]).
```

Comment:
Follow up test262 work item: test that iterator values are instants here (current test262 only checks that toString matches)

Reply:
Do you mean, test that in all built-in time zones the elements of the array returned from getPossibleInstantsFor are Instants?

Reply:
and that returning a non-Instant from getPossibleInstantsFor properly throws at this point, that is: when consuming the iterable